pkg/cli/common.go

```diff
@@ -42,6 +42,12 @@ type NameSpaceResults struct {
 	UTS           string
 }
 
+// PodmanBuildResults represents the results for Podman Build flags
```

Comment:
Can't we just add this to Podman?
libpod/cmd/build.go?

Reply:
Probably.  All the other definitions are here, so thought I'd keep them together.  I'll try adding it to cmd/build.go.

Reply:
Per @rhatdan's suggestion, I've moved this code into Podman in https://github.com/containers/libpod/pull/4215.  If those changes are merged there, I will close this PR.